app/(main)/settings/team/team-members-list-item.tsx

```diff
@@ -29,23 +29,22 @@ type TeamMemberListItemProps = {
 	displayName: string | null;
 	email: string | null;
 	role: TeamRole;
-	currentUserRoleState: [TeamRole, (role: TeamRole) => void];
```

Comment:
The parent component was introduced with the assumption that `currentUserRole` would be updated, but since re-rendering is now managed by key, this is no longer necessary, so `currentUserRole` is now used directly.